### PR TITLE
NAS-108991 / 21.02 / Add ability to update container images in a chart release

### PR DESCRIPTION
**Pull request by sonicaj**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py [hunk 1]
```diff
@@ -33,10 +33,16 @@ async def query(self, filters=None, options=None):
         `options.extra.history` is a boolean when set will retrieve all chart version upgrades for a chart release.
         """
         if not await self.middleware.call('service.started', 'kubernetes'):
-            return []
+            return filter_list([], filters, options)
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py:36]: Why?

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py:36]: Basically `get: true` fails where consumer would expect an error to be raised if there is no chart release but is instead presented with an empty list

Comment (william-gr) [reply src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py:36]: May I suggest a comment? Anyone looking at the code would probably wonder why we are filtering an empty list

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py:36]: Of course, adding it right away

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py:37]: Added :)

**Review by william-gr — APPROVED** [review]

**Pull request merged**